packages/mattermost-redux/src/actions/timezone.ts

```diff
@@ -28,3 +32,9 @@ export function autoUpdateTimezone(deviceTimezone: string) {
         }
     };
 }
+
+export function getSupportedTimezones() {
+    return async (dispatch: DispatchFunc) => {
+        dispatch({type: GeneralTypes.SUPPORTED_TIMEZONES_RECEIVED, data: timezones});
```

Comment:
It's a bit bigger of a change, but if this list is going to be hardcoded now, couldn't we remove it from redux entirely?

Reply:
Yeah that's true, I can do that

Reply:
Done, removed timezones from redux